pkg/cli/server/root_test.go

```diff
@@ -1256,11 +1262,11 @@ storage:
 		err = tmpCredsFile.Close()
 		So(err, ShouldBeNil)
 
-		content = []byte(fmt.Sprintf(`{ "distSpecVersion": "1.1.0-dev", 
-			"storage": { "rootDirectory": "/tmp/zot" }, "http": { "address": "127.0.0.1", "port": "8080", 
-			"auth": { "ldap": { "credentialsFile": "%v", "address": "ldap.example.org", "port": 389, 
-			"startTLS": false, "baseDN": "ou=Users,dc=example,dc=org", 
-			"userAttribute": "uid", "userGroupAttribute": "memberOf", "skipVerify": true, "subtreeSearch": true }, 
+		content = []byte(fmt.Sprintf(`{ "distSpecVersion": "1.1.0-dev",
```

Comment:
Are these whitespace cleanups? Would move this to a separate commit

Reply:
I've removed these changes